k8s/pkg/apis/longhorn/v1beta1/backup.go

```diff
@@ -11,42 +11,62 @@ const (
 	BackupStateUnknown    = BackupState("Unknown")
 )
 
+// SnapshotBackupSpec defines the desired state of the Longhorn backup
 type SnapshotBackupSpec struct {
+	// The time to request run sync the remote backup.
 	// +optional
 	SyncRequestedAt metav1.Time `json:"syncRequestedAt"`
+	// The snapshot name.
 	// +optional
 	SnapshotName string `json:"snapshotName"`
+	// The labels of snapshot backup.
 	// +optional
 	Labels map[string]string `json:"labels"`
 }
 
+// SnapshotBackupStatus defines the observed state of the Longhorn backup
 type SnapshotBackupStatus struct {
+	// The owner ID which is responsible to reconcile this backup CR.
```

Comment:
`The node ID on which the controller is responsible to reconcile this backup volume CR.`

similar below and above.